trinity/protocol/les/peer.py

```diff
@@ -163,7 +175,7 @@ def __init__(self,
             SendBlockHeadersEvent,
             lambda ev: self.try_with_session(
                 ev.session,
-                lambda peer: peer.sub_proto.send_block_headers(ev.headers, ev.buffer_value, ev.request_id)  # noqa: E501
+                lambda peer: peer.sub_proto.send(ev.command)
```

Comment:
Any reason we need to leave this as sub_proto? vs...

```suggestion
                lambda peer: peer.les_api.send(ev.command)
```